src/Features/Core/CodeFixes/CodeFixService.cs

```diff
@@ -106,7 +106,7 @@ internal partial class CodeFixService : ICodeFixService
             }
         }
 
-        public async Task<IEnumerable<CodeFixCollection>> GetFixesAsync(Document document, TextSpan range, CancellationToken cancellationToken)
+        public async Task<IEnumerable<CodeFixCollection>> GetFixesAsync(bool includeSuppressionFixes, Document document, TextSpan range, CancellationToken cancellationToken)
```

Comment:
Minor: Probably should be the second last param, so that all the callsites can use a named argument for it.